crates/python/qcs_sdk/__init__.pyi

```diff
@@ -200,11 +226,11 @@ class RegisterMatrix:
     def to_real(self) -> NDArray[np.float64]: ...
     def to_complex(self) -> NDArray[np.complex128]: ...
     @staticmethod
-    def from_integer(inner: NDArray[np.int64]) -> "RegisterMatrix": ...
+    def from_integer(matrix: NDArray[np.int64]) -> "RegisterMatrix": ...
```

Comment:
I see per https://github.com/rigetti/qcs-sdk-rust/blob/55f28f7de80cfa18074429720d3b788d6bf91ebd/crates/python/src/execution_data.rs#L136 that these are implemented manually instead of with `pyo3` macros - is there a reason to use the keyword `matrix` here instead of aligning with the macro's use of `inner` ?

Reply:
No good reason, just that `stubtest` flagged it now that `__text_signature__` is defined by default. Will change it to inner.